server/routes/maintainingReports/coordinator.test.ts

```diff
@@ -390,4 +393,104 @@ describe('coordinator', () => {
+    describe('viewRemovalRequest', () => {
+      it('should call involvedStaffService and userService', async () => {
+        involvedStaffService.loadInvolvedStaff.mockResolvedValue({
+          statementId: 1,
+          name: '',
+          userId: 'someUserId',
+          email: '',
+        })
+        involvedStaffService.getInvolvedStaffRemovalRequestedReason.mockResolvedValue('')
+        userService.getUserLocation.mockResolvedValue('Leeds')
+        userSupplier.mockReturnValue(coordinatorUser)
+
+        await request(app)
+          .get('/coordinator/report/123/statement/2/view-removal-request')
+          .expect(200)
+          .expect(res => {
+            expect(res.text).toContain('Request to be removed from use of force incident')
+          })
+
+        expect(involvedStaffService.loadInvolvedStaff).toBeCalledWith(123, 2)
+        expect(involvedStaffService.getInvolvedStaffRemovalRequestedReason).toBeCalledWith(2)
+        expect(userService.getUserLocation).toBeCalledWith('user1-system-token', 'someUserId')
+      })
+    })
+
+    describe('submitRemovalRequest', () => {
+      it('should redirect to itself if yes no not selected', async () => {
+        userSupplier.mockReturnValue(coordinatorUser)
+        const flash = jest.fn().mockReturnValue([])
+
+        app = appWithAllRoutes(
+          {
+            involvedStaffService,
+            reportService,
+            offenderService,
+            reviewService,
+            userService,
+          },
+          userSupplier,
+          null,
+          flash
+        )
+
+        await request(app)
+          .post('/coordinator/report/123/statement/2/view-removal-request')
+          .send({ confirm: undefined })
+          .expect(302)
+          .expect('Location', '/coordinator/report/123/statement/2/view-removal-request')
+          .expect(() => {
+            expect(flash).toBeCalledWith('errors', [
+              {
+                text: 'Select yes if you want to remove this person from the incident',
+                href: '#confirm',
+              },
+            ])
+          })
+      })
+
+      it('should redirect to confirm-delete if yes selected', async () => {
+        userSupplier.mockReturnValue(coordinatorUser)
+        await request(app)
+          .post('/coordinator/report/123/statement/2/view-removal-request')
+          .send({ confirm: 'yes' })
+          .expect(302)
+          .expect('Location', '/coordinator/report/123/statement/2/confirm-delete')
+      })
+
+      it('should redirect to not-removed if no selected', async () => {
+        userSupplier.mockReturnValue(coordinatorUser)
+        await request(app)
+          .post('/coordinator/report/123/statement/2/view-removal-request')
+          .send({ confirm: 'no' })
+          .expect(302)
+          .expect('Location', '/coordinator/report/123/statement/2/staff-member-not-removed')
+      })
+    })
+
+    describe('staffmemberNotRemoved', () => {
```

Comment:
a few situations where using camel case for test names - these should be normal case

Reply:
Done